config/manager/manager.yaml

```diff
@@ -42,6 +42,8 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
         - name: ADDON_NAME
+        - name: SOP_ENDPOINT
+          value: "https://red-hat-storage.github.io/ocs-sop/sop/OSD/{{ .GroupLabels.alertname }}.html"
```

Comment:
This should be set in the CSV in the addon bundle not here
So the var here should be empty (same as ADDON_NAME)

Reply:
Why not have a value here? I very often forget to set the ADDON_NAME when deploying from the csv and would prefer not to have another configuration to remember.

Reply:
Two reasons:
1. This is an upstream project and as such should not point to a RedHat specific thing
2. I am not sure that the env var will be overridden if it is already set